### ISSUE TITLE
[Bug]: Incomplete source jar

### ISSUE THREAD

**Issue by Technici4n** [FTBTeam/FTB-Mods-Issues#918]
### Mod

FTB Teams

### Mod version

ftb-teams-fabric:2001.1.4

### Forge / Fabric version

loader 0.14.22 fapi 0.87.0+1.20.1

### What issue are you having?

Sources jar doesn't contain sources... :P

![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/13494793/bf490e9e-c5cc-4e7d-8d3d-25b33c8b7d63)

### Steps to reproduce

Note: I use Loom

**Comment by desht** [comment]
Yeah, the -forge and -fabric source jars aren't very useful, but the common JAR should have what you need: https://maven.ftb.dev/#/releases/dev/ftb/mods/ftb-teams/2001.1.4

But it would make sense to have the platform JARs have the appropriate corresponding source jars, so I'll take a look at that.

**Comment by Technici4n** [comment]
My workaround was to look at the source on GitHub, that is enough for what I need. :smile:
Thanks for looking into it.
